### DISCUSSION TITLE
Do Space scheduled actions run instantly?

### DISCUSSION THREAD

**Discussion by s4m-mo** [deta/space-docs#23] [Q&A] [answered]
Basically what the title says. When you have stuff in the `actions` part of your **Spacefile**, does the scheduled action run as soon as you `space push` or `space release`? Or does it wait until it's interval is done?

**Comment by abdelhai** [comment, marked as answer]
it doesn't run immediately, it runs at the nearest time (the scheduler maintains the reference, not the app itself)

**Reply by s4m-mo** [reply]
Thanks
